src/cmakeProject.ts

```diff
@@ -876,11 +876,13 @@ export class CMakeProject {
                         fullPath: file
                     })) : [];
                     const browse: string = localize("browse.for.cmakelists", "[Browse for CMakeLists.txt]");
-                    items.push({ label: browse, fullPath: "", description: "Search for CMakeLists.txt on this computer" });
+                    const dontAskAgain: string = localize("do.not.ask.again", "[Do not ask again]");
```

Comment:
Nit: Is the common convention "Don't Ask Again" in other places in VSCode?

Reply:
The usage in VS Code and the C/C++ extension is "Don't Show Again", see https://github.com/microsoft/vscode/blob/0991b0053f7664e5d88bb0b150822e6cc606a25a/src/vs/workbench/services/notification/common/notificationService.ts#L289 .

Reply:
Changing.